pkg/activator/handler/handler.go

```diff
@@ -61,7 +72,47 @@ func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	err := a.Throttler.Try(revID, func() {
-		attempts, httpStatus := a.proxyRequest(w, r, target)
+		var attempts int
+		var httpStatus int
+		// If a GET probe interval has been configured, then probe
+		// the queue-proxy with our network probe header until it
+		// returns a 200 status code.
+		success := (a.GetProbeCount == 0)
+		for attempts = 0; attempts < a.GetProbeCount; attempts++ {
```

Comment:
```suggestion
		for attempts = 0; attempts < a.GetProbeCount && !success; attempts++ {
```
And remove break below?

Reply:
Done